.buildkite/others.rayci.yml

```diff
@@ -17,12 +17,14 @@ steps:
         --only-tags doctest
         --except-tags gpu
         --parallelism-per-worker 3
+        --upload
```

Comment:
why is this required? why only for doctest?

Reply:
it's for controlling the rollout of this feature instead of rolling out for every test jobs